hack/cherry_pick_pull.sh

```diff
@@ -168,6 +170,9 @@ for pull in "${PULLS[@]}"; do
       fi
     done
 
+    # remove the patch file from /tmp
+    rm -f "/tmp/${pull}.patch"
```

Comment:
oh right yeah i moved it outside the {} and not to `return to kansas` because we would have to iterate the pulls loop again